libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -233,6 +233,32 @@ void GCS_MAVLINK::send_power_status(void)
                                   hal.analogin->power_status_flags());
 }
 
+void GCS_MAVLINK::send_battery_status(const AP_BattMonitor &battery, const uint8_t instance) const
```

Comment:
Header calls the second argument _status_, here it is _instance_. Can you uniformize the two please?